blackmarket/js/prestige.js

```diff
@@ -1,36 +1,36 @@
-    new PrestigeRank("Bronze Master",         	3,  1000,       		8.88),
-    new PrestigeRank("Silver I",				4,  2500,       		20.00),
-    new PrestigeRank("Silver II",            	5,  10000,      		30.00),
-    new PrestigeRank("Silver III",           	6,  25000,      		40.00),
-    new PrestigeRank("Silver Master",         	7,  50000,     			50.00),
-    new PrestigeRank("Gold I",					8,  100000,     		75.00),
-    new PrestigeRank("Gold II",          		9,  250000,    			100.00),
-    new PrestigeRank("Gold III",         		10, 1000000,			125.00),
-    new PrestigeRank("Gold Master",				11, 5000000,			150.00),
-    new PrestigeRank("Diamond I",				12, 17500000, 			200.00),
-    new PrestigeRank("Diamond II",  			13, 50000000,   		300.00),
-    new PrestigeRank("Diamond III",  			14, 100000000,  		400.00),
-    new PrestigeRank("Diamond Master",			15,	250000000,			500.00),
-    new PrestigeRank("Platinum I",				16,	500000000,			750.00),
-    new PrestigeRank("Platinum II",				17,	1000000000,			1000.00),
-    new PrestigeRank("Platinum III",			18,	2000000000,			1250.00),
-    new PrestigeRank("Platinum Master",			19,	5000000000,			1500.00),
-    new PrestigeRank("Legendary Dealer I",		20,	10000000000,		2000.00),
-    new PrestigeRank("Legendary Dealer II",		21,	25000000000,		3000.00),
-    new PrestigeRank("Legendary Dealer III",	22,	50000000000,		4000.00),
-    new PrestigeRank("Legendary Dealer IV",		23,	100000000000,		5000.00),
-    new PrestigeRank("Legendary Dealer V",		24,	200000000000,		6000.00),
-    new PrestigeRank("Legendary Dealer Master",	25,	500000000000,		7000.00),
-    new PrestigeRank("Hacker I",				26,	1000000000000,		9000.00),
-    new PrestigeRank("Hacker II",				27,	1500000000000,		11000.00),
-    new PrestigeRank("Hacker III",				28,	3000000000000,		13000.00),
-    new PrestigeRank("Hacker IV",				29,	7500000000000,		15000.00),
-    new PrestigeRank("Hacker V",				30,	20000000000000,		17000.00),
-    new PrestigeRank("Hacker Master",			31,	50000000000000,		19000.00)
+    new PrestigeRank("Street Rat I",				0,  25,         		2.22),
+    new PrestigeRank("Street Rat II",            	1,  100,        		4.44),
+    new PrestigeRank("Street Rat III",           	2,  400,        		6.66),
+    new PrestigeRank("Petty Thief I",         	3,  1000,       		8.88),
+    new PrestigeRank("Petty Thief II",				4,  2500,       		20.00),
+    new PrestigeRank("Petty Thief III",            	5,  10000,      		30.00),
+    new PrestigeRank("Dealer I",           	6,  25000,      		40.00),
+    new PrestigeRank("Dealer II",         	7,  50000,     			50.00),
+    new PrestigeRank("Dealer III",					8,  100000,     		75.00),
+    new PrestigeRank("Prospective Supplier",          		9,  250000,    			100.00),
+    new PrestigeRank("Drug Supplier I",         		10, 1000000,			125.00),
+    new PrestigeRank("Drug Supplier II",				11, 5000000,			150.00),
+    new PrestigeRank("Drug Supplier III",				12, 17500000, 			200.00),
```

Comment:
I realized that the earlier version had a lot of I, II, III, Master, I, II, II etc etc.
I figured from this point on that I would name them all individually. Hope you like it.